tests/NelmioSolariumExtensionTest.php

```diff
@@ -411,7 +415,6 @@ public function testLoadBalancer()
         /** @var Loadbalancer $loadBalancerPlugin */
         $loadBalancerPlugin = $client->getPlugin('loadbalancer');
         $this->assertInstanceOf(Loadbalancer::class, $loadBalancerPlugin);
-        $this->assertEquals($loadBalancerPlugin, $container->get('solarium.client.client1.load_balancer'));
```

Comment:
Why removing this? is the service private and the container do not allows to fetch it anymore?

Reply:
Exactly